taqo_client/lib/widgets/app_drawer.dart

```diff
@@ -135,6 +155,31 @@ class _TaqoAppDrawerWidget extends StatelessWidget {
             },
             enabled: authProvider.isAuthenticated,
           ),
+
+          // Added for bug #72
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 16),
+            child: Divider(),
+          ),
+          //AssetImage('assets/sentiment_very_dissatisfied.png'),
+          ListTile(
```

Comment:
Can you use `AboutListTile` documented at https://api.flutter.dev/flutter/material/AboutListTile-class.html? This way you don't need to define the `aboutTaqoWidget`.

Reply:
Sorry, it was my mistake. Yes, please keep the additional variable, while using `AboutListTile`. Thanks!